tools/marvin/marvin/lib/common.py

```diff
@@ -334,13 +335,57 @@ def get_template(
         return FAILED
 
     for template in list_templatesout:
-        if template.isready and template.templatetype == template_type:
+        if template.isready and template.templatetype == template_type and template.name.startswith('CentOS'):
             return template
     '''
     Return default first template, if no template matched
     '''
     return list_templatesout[0]
 
+def get_template_macchinina(
```

Comment:
This function and the ``get_template`` function seem very similar.  The only difference appears to be that  ``get_template`` function is checking for ``CentOS`` and ``get_template_macchinia`` is checking for ``macchinia``.  Please refactor these functions to remove the duplication.